vc_infer_pipeline.py

```diff
@@ -89,6 +89,7 @@ def get_f0(
         semitone_shift,
         f0_method,
         filter_radius,
+        auto_correct_pitch,

```

Comment:
Please move this after semitone_shift, so the parameters are bunched together by theme.

Reply:
Done